src/main/java/com/avispl/symphony/dal/infrastructure/management/samsung/smartthings/common/AggregatedDeviceControllingMetric.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022 AVI-SPL, Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.infrastructure.management.samsung.smartthings.common;
+import java.util.Arrays;
+import java.util.Optional;
+
+/**
+ * Aggregated Device Controlling Metric
```

Comment:
Ok this isn't what I meant exactly :) This description is sort of done by the class name already, what I mean is... Javadoc description that would explain what purpose does it serve exactly for someone who's out of context (like Bohdan, who was asked to review adapters code before, and who has never seen what adapters really are).

Reply:
I fixed it